src/assets/templates/academics_semesters.html

```diff
@@ -36,9 +36,9 @@ <h4 class="cc-left" data-ng-if="semester.notation" data-ng-bind="semester.notati
           </thead>
           <tbody data-ng-repeat="class in semester.classes">
               <tr data-ng-repeat-start="section in class.sections" data-ng-if="section.is_primary_section && !section.waitlisted && !class.transcript">
-                <td>
+                <td class="cc-academics-semester-course-title">
```

Comment:
@redconfetti can you simply use `cc-text-bold` here, instead of creating the new class?